assets/helpers/paymentIntegrations/stripeCheckout.js

```diff
@@ -82,8 +108,3 @@ const openDialogBox = (amount: number, email: string) => {
     });
   }
 };
-
-export {
```

Comment:
We tend to prefer to export an object at the bottom of the file over having exports along the file.